README.md

```diff
@@ -0,0 +1,212 @@
+# OpenIddict, as an OpenID Connect Addict
+
+### What's OpenIddict ?
+
+OpenIddict library aims to provide a simple and easy out of the box solution 
+to implement an OpenID Connect server for ASP.NET 5.
+
+It leverages the use of Identity (for user management) and 
+EntityFramework (as a store provider).
+
+Under the hood it uses [AspNet.Security.OpenIdConnect.Server](https://github.com/PinpointTownes/AspNet.Security.OpenIdConnect.Server) 
```

Comment:
The sentence sounds weird (and the link should point to aspnet-contrib, not to my fork :smile:)